Source/ACE.Server/WorldObjects/Player.cs

```diff
@@ -49,6 +49,7 @@ public Player(Weenie weenie, ObjectGuid guid, uint accountId) : base(weenie, gui
             Character.Id = guid.Full;
             Character.AccountId = accountId;
             Character.Name = GetProperty(PropertyString.Name);
+            Character.IsPlussed = true;
```

Comment:
I don't believe setting IsPlussed = true here to be correct.

Can you describe steps to reproduce the + issue that you're seeing?

Reply:
I just did some more testing and IsPlussed should be set to 'false' at this point. 

The issue I saw was with the latest server code, creating a new character, On login the Sever would throw an error. 
"SaveCharacter failed with exception: Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details. ---> MySql.Data.MySqlClient.MySqlException: Field 'is_Plussed' doesn't have a default value ---> MySql.Data.MySqlClient.MySqlException: Field 'is_Plussed' doesn't have a default value"